packages/typespec-ts/tsconfig.json

```diff
@@ -5,8 +5,8 @@
     "composite": true,
     "lib": ["ES2019"],
     "target": "es2019",
-    "module": "NodeNext",
-    "moduleResolution": "NodeNext",
+    "module": "node16",
+    "moduleResolution": "node16",
```

Comment:
Change to this file shouldn't actually be needed if you prefer to stay to `nodenext`